content/en/security/account_takeover_protection.md

```diff
@@ -0,0 +1,268 @@
+3. Notifications. Ensures you are notified of compromised accounts.
+4. Reviewing your first detection. Understand how automated protection fits in with your attacks and escalation requirements.
+
+
+## Instrumenting your production login endpoints
+
+The following user activity events are used for ATO tracking.
+
+| Enrichment              | Auto-instrumented | Use case                                     |
+|-------------------------|-------------------|----------------------------------------------|
+| `users.login.success`     | True              | Account takeover detection rule requirement       |
+| `users.login.failure`     | True              | Account takeover detection rule requirement       |
+| `users.exists`            | False             | Signal review and determining if attack is targeted |
+| `user.password_reset`     | False             | Detection rule requirement to identify user enumeration through password reset |
+
+For steps on enabling tracking for events that are not automatically instrumented, go to [User Monitoring and Protection][1].
+
+For the latest list of relevant detections and instrumentation requirements, go to [Detection Rules][2] page.
+
+
+[Automatic instrumentation][3] is a Datadog capability that automatically identifies user login success and failure for most authentication implementations. We recommend that applications are additionally instrumented for all recommended enrichments, such as `users.exists`.
+
+You are not limited to how Datadog defines these enrichments. Many platform products opt to add additional enrichments, such as identifying the customer organization or user role.
+
+## Remote Configuration
+
+[Remote Configuration][4] enables ASM users to instrument apps with enrichments such as `users.exists` or custom [business logic][5] data in near real time.
+
+
+## Notifications
+
+[Notifications][6] are a flexible method to ensure the correct team members are informed of an attack. Collaboration [Integrations][7] with common communication methods are available out of the box.
+
+
+## Review your first detection
+
+ASM highlights the most relevant information and suggests actions to take based on the detection type. It also indicates what actions have been taken.
+
+{{<img src="security/ato/review_first_detection2.png" alt="An Account Takeover signal showing different highlighted areas of interest" style="width:100%;">}}
+
+**Compromised Users**
+
+Compromised and targeted users can be reviewed and blocked within **Signals** and **Traces**.
+
+**Signals**
+
+Individual users can be blocked in **Signals** using **Targeted Users**.
+
+{{<img src="security/ato/compromised_users_signals2.png" alt="Compromised users shown on a security signal" style="width:100%;">}}
+
+**Traces**
+
+Individual users can be blocked on **Traces** in **User**. Click on any user to show this option.
```

Comment:
in the **User drawer**

(the user drawer is what you see when you click on a user)